test/fixtures/fixtures.js

```diff
@@ -1179,11 +1179,7 @@ export function getTargetingKeys() {
     ],
     [
       'foobar',
-      '300x250'
-    ],
-    [
-      'foobar',
-      '300x250'
+      ['0x0', '300x250', '300x600']
```

Comment:
why is 0x0 being set?

Reply:
This is getting set because of a scenario like this https://github.com/prebid/Prebid.js/issues/1574#issuecomment-335911925

In fixture 
https://github.com/prebid/Prebid.js/blob/master/test/fixtures/fixtures.js#L319
https://github.com/prebid/Prebid.js/blob/master/test/fixtures/fixtures.js#L348

```
    {
      'bidderCode': 'triplelift',
      'adUnitCode': '/19968336/header-bid-tag-0',
      'adserverTargeting': {
        'foobar': '0x0'
      }
    },
    {
      'bidderCode': 'appnexus',
      'adUnitCode': '/19968336/header-bid-tag-0',
      'adserverTargeting': {
        'foobar': '300x250'
      }
    }
```